packages/block-editor/src/components/iframe/index.js

```diff
@@ -151,6 +153,10 @@ function setBodyClassName( doc ) {
 	doc.dir = document.dir;
 	doc.body.className = BODY_CLASS_NAME;
 
+	dispatch( blockEditorStore ).__unstableSetIframedEditorCanvasWrapper(
+		doc.body
+	);
```

Comment:
What about doing this in its own separate function?
I don't see much relation with `setBodyClassName`. 🤔